driver-pulsar/src/main/java/io/nosqlbench/driver/pulsar/ops/PulsarAdminTenantOp.java

```diff
@@ -45,7 +45,8 @@ public void run() {
 
         // Admin API - create tenants and namespaces
         if (!adminDelOp) {
-            TenantInfo tenantInfo = new TenantInfo();
+
+            TenantInfoImpl tenantInfo = TenantInfoImpl.builder().build();
```

Comment:
This should be TenantInfo.builder() probably and we should not use the Impl type